README.md

```diff
@@ -42,6 +42,24 @@ In any case, the following binaries will be created in the `bin` subfolder:
 * `vcf_debugulator`: automatic fixing tool
 * `test_validator` and derivatives: testing correct behaviour of the tools listed above
 
+### ODB & Ragel build
```

Comment:
"Developers build"? That would prevent some end users to even look at this section without any further clarification, and if we have some other dependencies in the future the title is already generic.